src/shared/services/MirrorNodeService/MirrorNodeService.ts

```diff
@@ -263,7 +263,8 @@ function createMirrorNodeService() {
     const allEvents = decodeLog(abiSignatures, response, ["ProposalDetails"]);
     const proposalCreatedEvents = allEvents.get("ProposalDetails") ?? [];
     const proposals: MirrorNodeDecodedProposalEvent[] = proposalCreatedEvents.map((item: any) => {
-      return { ...item, contractId, type: proposalType };
+      // TODO: added the conversion from BigInt to String. Remove it with making changes in the Interface.
+      return { ...item, contractId, type: proposalType, proposalId: item.proposalId + "" };
```

Comment:
why this was required.?

Reply:
When we upgraded the Web3 version, it changed the decode logic. earlier proposalId was coming as string. Now it is decoding to  BigInt. So we needed one extra layer to convert is back to string.